mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+    return "Hello, my name is " + name + " and I am " + age + " years old";
+}
 
 function getTotal(a, b) {
-    total = a ++ b;
-
-    return "The total is total"
+    total = a + b;
```

Comment:
Hi Aitken! This is David, your edu buddy, how are ya? While this code works, keep in mind that this is a global variable. Declaring a global variable like this inside a function is discouraged. You can read this https://stackoverflow.com/questions/10525582/why-are-global-variables-considered-bad-practice, or reach out to me if you have more questions.